api/controllers/AuthController.js

```diff
@@ -1,175 +1,35 @@
-'use strict';
-
-var async = require('async');
-var _ = require('lodash');
-
-/**
- * Authentication Controller
- *
- * This is merely meant as an example of how your Authentication controller should look. It currently
- * includes the minimum amount of functionality for the basics of Passport.js to work.
- */
-var AuthController = {
-  /**
-   * Log out a user and return them to the homepage
-   *
-   * Passport exposes a logout() function on request (also aliased as logOut()) that can be
-   * called from any route handler which needs to terminate a login session. Invoking logout()
-   * will remove the request.user property and clear the login session (if any).
-   *
-   * For more information on logging out users in Passport.js, check out:
-   * http://passportjs.org/guide/logout/
-   *
-   * @param   {Request}   request     Request object
-   * @param   {Response}  response    Response object
-   */
-  logout: function logout(request, response) {
-    request.logout();
-
-    response.json(200, true);
-  },
-
-  /**
-   * Create a third-party authentication endpoint
-   *
-   * @param   {Request}   request     Request object
-   * @param   {Response}  response    Response object
-   */
-  provider: function provider(request, response) {
-    sails.services['passport'].endpoint(request, response);
-  },
-
-  /**
-   * Simple action to check current auth status of user. Note that this will always send
-   * HTTP status 200 and actual data will contain either user object or boolean false in
-   * cases that user is not authenticated.
-   *
-   * @todo    Hmmm, I think that this will return always false, because of missing of
-   *          actual sessions here...
-   *
-   * @param   {Request}   request     Request object
-   * @param   {Response}  response    Response object
-   */
-  authenticated: function authenticated(request, response) {
-    if (request.isAuthenticated()) {
-      response.json(200, request.user);
-    } else {
-      response.json(200, false);
+var passport = require('passport');
+
```

Comment:
We will work in a script to format all the source code with 2 spaces in the identation